BuildingSync.xsd

```diff
@@ -796,6 +796,18 @@
           </xs:simpleContent>
         </xs:complexType>
       </xs:element>
+      <xs:element name="TotalExteriorRoofArea" minOccurs="0">
```

Comment:
Wondering if we should use `TotalRoofArea` instead? I don't have a preference, was wondering if there might have been a reason we added `Exterior` to the wall area one, but if it is necessary for roofs.

Reply:
I think total roof area is concise and better. Is there a non-exterior/interior roof anywhere?

Reply:
I think `TotalRoofArea` is good.